EnhancedMediaPlayer/SettingsMenu.swift

```diff
@@ -0,0 +1,39 @@
+// SettingsMenu.swift
+import SwiftUI
+
+public struct SettingsMenu: View {
+    public init() { }
+
+    public var body: some View {
+        VStack(alignment: .leading, spacing: Constants.stackSpacing) {
+            renderSettingsItem()
+            renderSettingsItem()
+            renderSettingsItem()
+        }
+        .padding(Constants.stackSpacing)
+        .background(.background)
+        .cornerRadius(Constants.cornerRadius)
+    }
```

Comment:
I feel like this menu items should stick to the top of the thing that comes up...